electrochemicalquantities.ttl

```diff
@@ -138,10 +132,14 @@ skos:prefLabel rdf:type owl:AnnotationProperty .
 
 ###  http://emmo.info/electrochemistry#electrochemistry_09e64707_a17d_4405_84cc_ee9d91ed32ef
 :electrochemistry_09e64707_a17d_4405_84cc_ee9d91ed32ef rdf:type owl:Class ;
-                                                       rdfs:subClassOf :electrochemistry_4ebe2ef1_eea8_4b10_822d_7a68215bd24d ,
-                                                                       :electrochemistry_c5047d29_4e68_43ee_8355_3e8f05dc70a5 ;
+                                                       rdfs:subClassOf :electrochemistry_c5047d29_4e68_43ee_8355_3e8f05dc70a5 ,
+                                                                       [ rdf:type owl:Restriction ;
+                                                                         owl:onProperty emmo:EMMO_bed1d005_b04e_4a90_94cf_02bc678a8569 ;
+                                                                         owl:someValuesFrom emmo:EMMO_6837b106_7220_4ec6_b7c9_d549d6163672
+                                                                       ] ;
                                                        emmo:EMMO_967080e5_2f42_4eb2_a3a9_c58143e835f9 "In CCCV methods, the cell voltage at which the constant current mode switches to constant voltage mode."@en ;
                                                        emmo:EMMO_c7b62dd7_063a_4c2a_8504_42f7264ba83f "The imposed constant voltage is often the same value as the voltage change limit."@en ;
+                                                       skos:altLabel "dV/dt"@en ;
```

Comment:
Yes, the elucidation should make that more clear.

Reply:
Really though, the elucidation doesn't make much sense. The switch from CC to CV is done by a voltage limit rather than a dV/dt limit. The quantity itself is still useful, but I will change the elucidation to make it a more general term.